src/libs/ReportUtils.ts

```diff
@@ -6506,7 +6507,7 @@ function buildOptimisticEditedTaskFieldReportAction({title, description}: Task):
             {
                 type: CONST.REPORT.MESSAGE.TYPE.COMMENT,
                 text: changelog,
-                html: getParsedComment(changelog),
+                html: getParsedComment(changelog, undefined, undefined, title !== undefined ? [...CONST.TASK_TITLE_DISABLED_RULES] : undefined),
```

Comment:
I am not sure why we want to spread (`...`) the value and then pass it as array `[]`. Why not just pass `CONST.TASK_TITLE_DISABLED_RULES` ?